### ISSUE TITLE
Add status column to service binding

### ISSUE THREAD

**Issue by PK85** [kyma-project/kyma#1593]
As a developer I like to see if service bindings are correctly created. Right now Lambda UI do not display service binding is they are in error state. For example, service instance is not bindable.

AC:
- The status is shown in the list of bound services.

**Related PR(s)**
- https://github.com/kyma-project/console/pull/340

**Comment by joek** [comment]
@PK85 during review we figured out, that service binding status might not be the right thing to display as the more important part for the customer is the service binding usage. Can you please explain how the status of SB and SBU are related to each other and which one to display in lambda ui?

**Comment by PK85** [comment]
Hello @joek,

As a vision I'd like to have one UI component (reactJs + ui-api-layer) for that service binding view for all types of applications, and Function view should include that as a first app. We already discussed ui modal thing some days ago. That will allow us to apply changes only once in that base component and function view will get it out of the box.

I'm not sure if we want to continue that separate UI Lambda implementation in area of service bindings.

How that UI component should look like is a matter additional meeting, also with UI/UX taking into account all UI/UX we already added into service catalog UI.

Cheers, PK
